src/node.cpp

```diff
@@ -984,6 +984,40 @@ NodeCounter Node::subnodeCounts() const
     return nc;
 }
 
+#ifdef ENABLE_SYNC
+void Node::setSyncable(const bool syncable, const std::string& syncPath)
+{
+    if (!syncable)
+    {
+        if (!client->unsyncables->addNode(nodehandle, syncPath))
+        {
+            LOG_err << "Incomplete database write for node: " << nodehandle;
+        }
+    }
+    else
+    {
+        if (!client->unsyncables->removeNode(nodehandle))
+        {
+            LOG_err << "Incomplete database write for node: " << nodehandle;
+        }
+    }
+}
+
+bool Node::isSyncable() const
+{
+    const auto syncPath = client->unsyncables->syncPath(nodehandle);
+    if (syncPath)
```

Comment:
wouldn't it be more efficient to test if the nodehandle is in the unordered_map?  thanks

Reply:
yes, fixed now.